.travis.yml

```diff
@@ -5,6 +5,18 @@ jdk:
   - oraclejdk8
 sudo: required
 dist: trusty
+cache:
+  directories:
+    - $HOME/mvn-home
+
+install:
+  # install maven 3.3.9 if it is not in cache already
```

Comment:
Pushed fixes for both of those. Note that 3.2.5 is the actual minimum maven version required.